src/rpdk/core/project.py

```diff
@@ -1337,7 +1348,12 @@ def _setup_stack_template_environment(self) -> None:
         )
         bootstrap_file = stack_template_root / CANARY_DEPENDENCY_FILE_NAME
         if dependencies_file.exists():
+            LOG.debug("Writing: %s", bootstrap_file)
```

Comment:
logs for writing files are set at debug level. this is consistent with other write operations in this file.